Rhinoceros_oM/BHoMRhinoObject.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BH.oM.Base;
+using BH.oM.Geometry;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.oM.Adapters.Rhinoceros
+{
+    public class BHoMRhinoObject : BHoMObject
+    {
+        /*******************************************/
+        /**** Properties                        ****/
+        /*******************************************/
+        public virtual string Layer { get; set; } = "";
+
+        public virtual IGeometry Geometry { get; set; } = null;
+
+        public virtual Color LayerColour { get; set; } = new Color();
```

Comment:
Would consider adding a `Layer` class as well to handle the layer colour and name. Then this object could just store a ref to that.

Reply:
Good idea. Added this.